src/main/java/gov/cms/madie/terminology/controller/VsacFhirTerminologyController.java

```diff
@@ -85,4 +86,13 @@ public ResponseEntity<Code> getCode(
     return ResponseEntity.ok()
         .body(fhirTerminologyService.retrieveCode(code, codeSystem, version, user.getApiKey()));
   }
+
+  @PostMapping(path = "/codesList", produces = MediaType.APPLICATION_JSON_VALUE)
+  public ResponseEntity<List<Code>> getCodesList(
+      @RequestBody() List<Map<String, String>> codeList, Principal principal) {
```

Comment:
Suggestion: Rename `codeList` to better explain what the Strings are in the Map. For instance, if they are the code and code system, then `codeAndCodeSystems` perhaps.

Reply:
thanks for the comment I have changed it to getCodesAndCodeSystems